ska_conda/ska_builder.py

```diff
@@ -10,43 +10,28 @@
 
 class SkaBuilder(object):
 
-    def __init__(self, build_root='.', user='sot', git_repo_path='git@github.com:{user}/{name}.git'):
-        self.user = user
-        self.git_repo_path = git_repo_path
+    def __init__(self, build_root='.'):
         self.build_dir = os.path.abspath(os.path.join(build_root, "builds"))
         self.src_dir = os.path.abspath(os.path.join(build_root, "src"))
         os.environ["SKA_TOP_SRC_DIR"] = self.src_dir
 
     def _clone_repo(self, name, tag=None):
         if name in no_source_pkgs:
             return
-        print("Cloning source %s." % name)
+        print("Cloning or updating source source %s." % name)
         clone_path = os.path.join(self.src_dir, name)
         if not os.path.exists(clone_path):
-            # Try ssh first to avoid needing passwords for the private repos
```

Comment:
Awesome!

Reply:
Regarding continuing, my thought was that there could be other reasons for failure that you might not care about if all you really wanted to do was update a couple of packages at the end of the list.  Or if you were a non-privileged user who'd just been handed this code base (JWST?) and this logic lets you really just see how everything works without stopping at parse_cm.  You could get in to dependency hell in a fresh build with no access to the currently build packages in ska3-conda, but the only fallout of this change in that case would just be that it would take a lot longer to finish failing out with the rest of the packages.

Reply:
Also, "forgetting ssh key password" was a creditable failure in my testing.

Reply:
OK, well then put in a text input to ask the user to continue ala `install.py`.

Reply:
OK.